gdx/src/com/badlogic/gdx/graphics/glutils/ETC1.java

```diff
@@ -114,10 +129,13 @@ public void write (FileHandle file) {
 					write.write(buffer, 0, bytesToWrite);
 					writtenBytes += bytesToWrite;
 				}
+
+				write.close();
+
+				gzip.close();
+
 			} catch (Exception e) {
-				throw new GdxRuntimeException("Couldn't write PKM file to '" + file + "'", e);
-			} finally {
-				StreamUtils.closeQuietly(write);

```

Comment:
Stream closing should still be in a finally block though. As you said, it is important to close it. And when it is there, closeQuietly has the benefit that it does the null check for you.